extensions/pooling/src/__tests__/acceptance/pooling.acceptance.ts

```diff
@@ -0,0 +1,370 @@
+// Copyright IBM Corp. 2020. All Rights Reserved.
+// Node module: @loopback/pooling
+// This file is licensed under the MIT License.
+// License text available at https://opensource.org/licenses/MIT
+
+import {
+  BindingKey,
+  Constructor,
+  Context,
+  ContextTags,
+  createBindingFromClass,
+  inject,
+  Provider,
+} from '@loopback/core';
+import {expect} from '@loopback/testlab';
+import {once} from 'events';
+import {Options} from 'generic-pool';
+import {
+  createPooledBindingFactory,
+  PoolingService,
+  PoolingServiceOptions,
+} from '../../';
+import {
+  getPooledValue,
+  Poolable,
+  PooledValue,
+  PoolFactory,
+} from '../../pooling';
+
+describe('Resource pool', () => {
+  const POOLING_SERVICE = BindingKey.create<PoolingService<ExpensiveResource>>(
+    'services.pooling',
+  );
+  let ctx: Context;
+
+  beforeEach(givenContext);
+
+  it('creates a resource pool', async () => {
+    const poolService = await givenPoolService();
+    expect(poolService.pool.size).to.eql(0);
+  });
+
+  it('creates a resource pool as singleton', async () => {
+    const poolService = await givenPoolService();
+    const result = await ctx.get(POOLING_SERVICE);
+    expect(result).to.be.exactly(poolService);
+  });
+
+  it('acquires/releases a resource from the pool', async () => {
+    const poolService = await givenPoolService({max: 5});
```

Comment:
Why not leave it out `{max: 5}`? It makes it look like the test has something to do with the parameter, when it is not. Same applies to other locations.

Reply:
Removed.